### DISCUSSION TITLE
Sort by default on data load

### DISCUSSION THREAD

**Discussion by miohtama** [grid-js/gridjs#963] [Q&A]
I am loading data to GridJs and it is in random order. Is there a way to define/trigger a default sort on load?

[I tried to find this in documentation, but did not figure it out](https://gridjs.io/docs/config/sort).

**Comment by nk9** [comment]
I'm looking for the same thing. It seems like a very common requirement, but I can't quite work out how to do it. This question [has been asked](https://stackoverflow.com/q/72543941/1749551) a [couple times](https://stackoverflow.com/q/66649688/1749551) on StackOverflow as well. There's also a [feature request](https://github.com/grid-js/gridjs/issues/1255).
